<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>audio</class>
 <widget class="QWidget" name="audio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="music_start">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>410</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>播放</string>
   </property>
  </widget>
  <widget class="QPushButton" name="music_pause">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>410</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>暂停/继续</string>
   </property>
  </widget>
  <widget class="QComboBox" name="music_comboBox">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>140</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="exitToMain">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>410</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QSlider" name="setVolume">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>170</y>
     <width>29</width>
     <height>160</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>291</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="current_time">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>360</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>00:00</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="all_time">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>366</y>
     <width>67</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>99:99</string>
   </property>
  </widget>
  <widget class="QSlider" name="singSlider">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>370</y>
     <width>221</width>
     <height>29</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="abc">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>390</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>XX%</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>140</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>340</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <zorder>exitToMain</zorder>
  <zorder>music_start</zorder>
  <zorder>music_pause</zorder>
  <zorder>music_comboBox</zorder>
  <zorder>setVolume</zorder>
  <zorder>listWidget</zorder>
  <zorder>current_time</zorder>
  <zorder>all_time</zorder>
  <zorder>singSlider</zorder>
  <zorder>abc</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton_3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
